packages/expo-video/CHANGELOG.md

```diff
@@ -20,6 +20,8 @@
 
 ### 💡 Others
 
+- [Android] Refactor `VideoPlayer.kt`, organise the files ([#30452](https://github.com/expo/expo/pull/30452) by [@behenate](https://github.com/behenate))
```

Comment:
```suggestion
- [Android] Refactor `VideoPlayer.kt`, organize files ([#30452](https://github.com/expo/expo/pull/30452) by [@behenate](https://github.com/behenate))
```

Reply:
Who came up with the idea of having British and American english instead of just one 😒